apiserver/facades/client/action/action.go

```diff
@@ -66,6 +66,7 @@ func newActionAPI(
 	getLeadershipReader func() (leadership.Reader, error),
 	applicationService ApplicationService,
 	blockCommandService common.BlockCommandService,
+	modelTag names.ModelTag,
```

Comment:
I suggest that we have passed in here the model uuid and construct the model tag internally where we need it based of this value.